app/__init__.py

```diff
@@ -375,6 +376,7 @@ def getRequestProjects():
         [ignore, recentDate] = getRecentDateAndRuns(project)
         project['ordering'] = recentDate
         project['date'] = time.strftime('%Y-%m-%d %H:%M', time.localtime((recentDate/1000)))
+        project['numComments'] = str(get_num_of_comments(project['requestId']))
```

Comment:
Can we pull 'str(get_num_of_comments(project['requestId']))' into a separate variable so we don't have to run this function twice